.travis.yml

```diff
@@ -1,19 +1,27 @@
 language: python
 python:
-    - "2.7"
+- '3.6'
```

Comment:
Minor grip but typically arrays in yaml are indented.

Reply:
In the files below (e.g. `.travis/2-master.yaml`) you're indenting array entries.